src/cljs/athens/views/pages/page.cljs

```diff
@@ -7,6 +7,18 @@
     [re-frame.core :as rf]))
 
 
+(defn page-by-title
+  []
+  (let [title           (rf/subscribe [:current-route/page-title])
+        {:keys [node/title
+                block/string
+                db/id]} @(pull db/dsdb '[*] [:node/title @title])]
```

Comment:
Pull will throw if the node title does not exist, and crash the app. This instead should do a e-by-av lookup, which does not throw.